maxim/tests/test_prompts.py

```
@@ -45,47 +38,66 @@ def tearDown(self):
+            QueryBuilder().and_().deployment_var("Environment", "prod").build(),
         )
         if prompt is None:
             raise Exception("Prompt not found")
+        self.assertEqual(prompt.prompt_id, promptId)
+        self.assertEqual(prompt.version_id, promptVersionId)
+        self.assertEqual(prompt.model, "gpt-3.5-turbo")
+        self.assertEqual(prompt.provider, "openai")
+        self.assertEqual(prompt.messages[0].content, "You are a helpful assistant")
+        self.assertEqual(len(prompt.messages), 1)
+        try:
+            resp = prompt.run(
```

Ruff (0.11.9)

Local variable `resp` is assigned to but never used

Remove assignment to unused variable `resp`

(F841)